src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -396,6 +406,30 @@ The **`stage`** denotes how this script (the `content`) should be inserted. Some
 
 A read-only copy of the user-supplied [Astro config](/en/reference/configuration-reference/). This is resolved _after_ other integrations have run.
 
+#### `setAdapter` option
+
+**Type:** `(adapter: AstroAdapter) => void;`
+
+Makes the integration an adapter. Read more in the [adapter API](/en/reference/adapter-reference/).
+
+#### `injectTypes` options
+
+<p><Since v="4.14.0" /></p>
+
+**Type:** `(injectedType: { filename: string; content: string }) => URL`
+
+Allows you to inject types easily in your user's project. `filename` must ends with `".d.ts"` and `content` must be valid TypeScript (it will be formatted).
```

Comment:
```suggestion
Allows you to inject types into your user's project. `filename` must ends with `".d.ts"` and `content` must be valid TypeScript.
```
- I'm not sure "in" vs "into" because it depends on what is actually happening here. But, if "into" doesn't sound "wrong" to you, then I suspect it's the correct one.

- The "it will be formatted for you" feels a *little* at odds with saying what you DO have to do. Does this suffer too much if we simply don't say this?

Reply:
- in vs into: it does a bunch of wiring under the hood so not sure what to use either! I think you know the nuance better so I trust you!
- It's not formatted anymore actually, the PR was updated